html/missing_page.html

```diff
@@ -9,40 +9,37 @@
     <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+    <link rel="stylesheet" src="https://cdn.datatables.net/1.10.21/css/dataTables.bootstrap4.min.css">
+    <script src="https://cdn.datatables.net/1.10.21/js/jquery.dataTables.min.js"></script>
+    <script src="https://cdn.datatables.net/1.10.21/js/dataTables.bootstrap4.min.js"></script>
     <script src="/samples/static/script.js" type="text/javascript"></script>
     <title>PdmV missing Samples for - {{ campaign_group }}*</title>
   </head>
   <body>
     <div >
       <div style="text-align: right; margin: 10px;">Logged in as {{ user_info.fullname }} ({{ user_info.role }})</div>
       <h5>PdmV missing Samples - {{ campaign_group }}</h5>
-      <div style="text-align: center; margin: 20px;">
-        <b>I am going to add or remove</b>
-        <select onChange="selectPWG(this, this.value)">
-          <option disabled selected value=''></option>
-          {% for pwg in pwgs %}
-            <option value="{{pwg}}">{{pwg}}</option>
-          {% endfor %}
-        </select>
-        <b>physics working group as interested PWG in samples</b>
-      </div>
 
       <p style="margin: 8px;">
+      {% set unique_rows = [] %}
```

Comment:
To me, this place (HTML template) looks like a wrong place to do these _not just drawing_ things. Why not pick unique short names in `missing_page` function and just print them here. It can be as simple as
```
short_names = sorted(list(set([r[0] for r in rows])))
```
and then just pass short names to `render_template()` function.